docs/standard/data/sqlite/transactions.md

```diff
@@ -25,7 +25,7 @@ A *nonrepeatable read* occurs when a transaction queries same row twice, but the
 
 *Phantoms* are rows that get changed or added to meet the where clause of a query during a transaction. If allowed, the same query could return different rows when executed twice in the same transaction.
```

Comment:
I'm gonna go with no hyphen since style guide says
Most words with common prefixes (such as co-, non-, pre-, re-, multi-, or un-) don’t include a hyphen.